cfme/cloud/stack.py

```diff
@@ -60,52 +66,69 @@ def edit_tags(self, tag, value):
-            sel.click(details_page.infoblock.element("Relationships", "Security Groups"))
-        if version.current_version() >= '5.5':
-            sel.click(details_page.infoblock.element("Relationships", "Security groups"))
-
-    def nav_to_parameters_link(self):
-        sel.force_navigate('clouds_stack', context={'stack': self})
-        sel.click(details_page.infoblock.element("Relationships", "Parameters"))
-
-    def nav_to_output_link(self):
-        sel.force_navigate('clouds_stack', context={'stack': self})
-        sel.click(details_page.infoblock.element("Relationships", "Outputs"))
-        cells = {'Key': "WebsiteURL"}
-        output_table().click_rows_by_cells(cells, "Key", True)
-
-    def nav_to_resources_link(self):
-        sel.force_navigate('clouds_stack', context={'stack': self})
-        sel.click(details_page.infoblock.element("Relationships", "Resources"))
-
     def wait_for_delete(self):
-        sel.force_navigate("clouds_stacks")
-        quad = Quadicon(self.name, 'stack')
-        wait_for(lambda: not sel.is_displayed(quad), fail_condition=False,
-            message="Wait stack to disappear", num_sec=500, fail_func=sel.refresh)
+        navigate_to(self, 'All')
+        wait_for(lambda: not sel.is_displayed(Quadicon(self.name, self.quad_name)),
+            fail_condition=False, message="Wait stack to disappear", num_sec=500,
+            fail_func=sel.refresh)
 
     def wait_for_appear(self):
-        sel.force_navigate("clouds_stacks")
-        quad = Quadicon(self.name, 'stack')
-        wait_for(sel.is_displayed, func_args=[quad], fail_condition=False,
-            message="Wait stack to appear", num_sec=1000, fail_func=sel.refresh)
-
-    def retire_stack(self):
-        sel.force_navigate('clouds_stack', context={'stack': self})
+        navigate_to(self, 'All')
+        wait_for(lambda: sel.is_displayed(Quadicon(self.name, self.quad_name)),
+            fail_condition=False, message="Wait stack to appear", num_sec=1000,
+            fail_func=sel.refresh)
+
+    def retire_stack(self, wait=True):
+        navigate_to(self, 'All')
+        sel.check(Quadicon(self.name, self.quad_name).checkbox())
         lifecycle_btn("Retire this Orchestration Stack", invokes_alert=True)
         sel.handle_alert()
         flash.assert_success_message('Retirement initiated for 1 Orchestration'
         ' Stack from the CFME Database')
-        sel.force_navigate("clouds_stacks")
-        quad = Quadicon(self.name, 'stack')
-        wait_for(lambda: not sel.is_displayed(quad), fail_condition=False,
-            message="Wait stack to disappear", num_sec=500, fail_func=sel.refresh)
+        if wait:
+            self.wait_for_delete()
+
+
+@navigator.register(Stack, 'All')
+class All(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def am_i_here(self):
+        title_match = sel.execute_script('return document.title;') == document_title
```

Comment:
believe this has been removed in favor of checking title and controller in a function and checking the summary_title() locally